interfaces/thirdparty/cp2k.py

```diff
@@ -140,25 +142,82 @@ def get_runtime(self):
         diff = endTime - startTime
         return diff.total_seconds()
 
-    def _get_energy_type(self, search='Total', index=0):
+    def get_timings(self):
+        """Return the Timings from the End of the Output File"""
+        chunk = self.get_output_chunk(begin="SUBROUTINE",end="-"*70)
+        ret = {}
+        for line in chunk[1:]:
+            line = line.split()
+            ret[line[0]] = line[1:]
+        return ret
+
+
+    def _get_energy_type(self, search='Total', index=-1, unit='a.u.'):
         if index:
             select = index
         else:
-            select = -1
-        s = self.grep_output(search + ' energy:')[select].split()[-1]
-        return float(s)
+            select = slice(0,None)
+
+        s = self.grep_output(search+' energy:')[select]
+        if not isinstance(select, slice):
+            s = [s]
+        return [ Units.convert(float(x.split()[-1]), 'a.u.', unit) for x in s ]
+
+    def get_energy(self, index=-1, unit='a.u.'):
+        """Returns 'Total energy:' from the output file.
+
+        Set ``index`` to choose the n-th occurence of the Charges in the output, e.g. to choose an optimization step.
+        Also supports slices.
+        Set to *None* to return all as a list.
+        Defaults to the last occurence.
+
+        Returns a List of Energies.
+        """
+        return self._get_energy_type('Total', index=index, unit=unit)
```

Comment:
I don't think returning a list of energies is a good idea here, as all the other `Job.get_energy()` methods return scalars.
Is it an idea to an additional keyword argument?

Reply:
True, changed the docstrings and the `_get_energy_type` returns.